### DISCUSSION TITLE
Run command shell on server from custom button

### DISCUSSION THREAD

**Discussion by monpelaud** [dimsemenov/PhotoSwipe#2065] [Q&A]
Hi,
I'm not a developer but I've been trying for quite some time to find a solution to run a shell command on the server from a custom button.
With this documentation https://photoswipe.com/adding-ui-elements/#adding-a-button-to-the-toolbar no problem to add a button to the Toolbar.
I'm stuck on the function associated with "onClick:". I'm looking for sample code to run a shell command on the server.
Could someone, who has already created this type of function, share their code?
Sorry if my request may seem strange to you but I am not a developer, just a newbie!

Thanks for your help.
Best regards.

**Comment by dimsemenov** [comment]
PhotoSwipe is a client-side component, it has no control of this. 

I suggest checking fetch API https://developer.mozilla.org/en-US/docs/Web/API/Fetch_API/Using_Fetch
